### PR TITLE
Add windows support to AWS

### PR DESCRIPTION
**Pull request by ehankland**

### CODE REVIEW

File: perfkitbenchmarker/aws/aws_virtual_machine.py [hunk 1]
```diff
@@ -328,3 +340,55 @@ def _GetDefaultImage(machine_type, region):
...
+        '--instance-id=%s' % self.id]
+    stdout, _ = util.IssueRetryableCommand(get_password_cmd)
+    response = json.loads(stdout)
+    password_data = response['PasswordData']
+    if not password_data:
+      raise ValueError('No PasswordData in response.')
+    return base64.b64decode(password_data)
+
+
+  def _PostCreate(self):
```
Comment (skschneider) [inline perfkitbenchmarker/aws/aws_virtual_machine.py]: Could you add a bit of explanation what's going on here?

File: perfkitbenchmarker/aws/aws_virtual_machine.py [hunk 2]
```diff
@@ -278,19 +285,24 @@ def CreateScratchDisk(self, disk_spec):
...
-      if self.local_disk_counter > self.max_local_disks:
-        raise errors.Error('Not enough local disks.')
-    else:
-      disks = [aws_disk.AwsDisk(disk_spec, self.zone)
-               for _ in range(disk_spec.num_striped_disks)]
+        if self.local_disk_counter > self.max_local_disks:
+          raise errors.Error('Not enough local disks.')
+      else:
+        # Remote disk numbers start at 1 + max_local disks (0 is the system disk
+        # and local disks occupy 1-max_local_disks).
```
Comment (skschneider) [inline perfkitbenchmarker/aws/aws_virtual_machine.py]: "1-max_local_disks" confused me (looked like subtraction). How about something like "[1, max_local_disks)" or expanding it out as a sentence?

File: perfkitbenchmarker/aws/aws_virtual_machine.py [hunk 3]
```diff
@@ -72,6 +74,8 @@
         SA_EAST_1: 'ami-6770d87a',
     }
 }
+HVM_US_EAST_1_WINDOWS_AMI = 'ami-cc93a8a4'
+WINDOWS = 'windows'
```
Comment (skschneider) [inline perfkitbenchmarker/aws/aws_virtual_machine.py]: Is this used?

File: perfkitbenchmarker/aws/aws_virtual_machine.py (hunk 1 above)
Comment (ehankland) [inline perfkitbenchmarker/aws/aws_virtual_machine.py]: Done

File: perfkitbenchmarker/aws/aws_virtual_machine.py (hunk 2 above)
Comment (ehankland) [inline perfkitbenchmarker/aws/aws_virtual_machine.py]: Done

File: perfkitbenchmarker/aws/aws_virtual_machine.py (hunk 3 above)
Comment (ehankland) [inline perfkitbenchmarker/aws/aws_virtual_machine.py]: Nope - removed - good catch

**Comment by ehankland** [comment]
PTAL

**Comment by skschneider** [comment]
LGTM

**Pull request merged**